tests/test_generate.py

```diff
@@ -27,33 +27,34 @@ class TestGenerate(TestCase):
     # some concurrency issues. If this fails, you can try rerunning the tests.
     @classmethod
     def tearDownClass(cls):
-        # Clear the fake2 directory of any test artifacts
-        cmd = "rm conf/locale/fake2/LC_MESSAGES/django.po; rm conf/locale/fake2/LC_MESSAGES/djangojs.po"
+        # Clear the mock locale directory of any test artifacts
+        fake_locale_dir = MOCK_APPLICATION_DIR / 'conf' / 'locale' / 'mock'
+        cmd = "rm {fake_locale_dir}/LC_MESSAGES/django.po; rm {fake_locale_dir}/LC_MESSAGES/djangojs.po".format(
+            fake_locale_dir=fake_locale_dir
+        )
         sys.stderr.write("\nCleaning up dummy language directories: {}\n".format(cmd))
         sys.stderr.flush()
         returncode = subprocess.call(cmd, shell=True)
```

Comment:
Yikes, I didn't realize we were using subprocess to delete files! :(  not your fault.... :)